src/op_mode/show_nat66_rules.py

```diff
@@ -40,7 +40,14 @@
     data_json = jmespath.search('nftables[?rule].rule[?chain]', tmp)
     for idx in range(0, len(data_json)):
         data = data_json[idx]
+        
+        # If there is no index 3, we don't think this is the record we need to check
+        if len(data['expr']) <= 3:
+            continue
+        
         comment = data['comment']
+        rule = comment.replace('SRC-NAT66-','')
```

Comment:
This looks very odd, what do you plan to acciebe hete?

Reply:
As the other script does, take the rule ID from the comment

Reply:
Why not use something like: https://stackoverflow.com/a/26825833

Reply:
For IPv4, NAT is OK, but if it is used for IPv6, it will result in unwanted numbers

Reply:
For nat only, nat66 is maintained